src/ui/Base/CoveoJQuery.ts

```diff
@@ -45,5 +51,5 @@ export function initCoveoJQuery() {
 }
 
 export function jQueryIsDefined(): boolean {
-  return window['$'] != undefined && window['$'].fn != undefined && window['$'].fn.jquery != undefined;
+  return window['$'] != undefined && window['$'].fn != undefined && window['$'].fn.jquery != undefined || window['Coveo'] != undefined && window['Coveo']['$'] != undefined;
```

Comment:
Add parenthesis around the two part of the || , and split on two line.

It becomes easier to read that we are checking either in global window scope, or local Coveo scope.